Makefile

```diff
@@ -4,11 +4,47 @@ PREFIX=/opt/yandex/mdb-ch-tools
 INSTALL_DIR=$(DESTDIR)$(PREFIX)
 
 .PHONY: install
-install: install-symlinks install-bash-completions install-logrotate ;
+install: install-python-package install-symlinks install-bash-completions configure-logs ;
 
 .PHONY: uninstall
 uninstall: uninstall-python-package uninstall-symlinks uninstall-bash-completions uninstall-logrotate ;
 
+.PHONY: install-python-package
+install-python-package: build-python-package
+	@echo 'Installing mdb-ch-tools'
+
```

Comment:
Why can't you rely on `flit install` here?

Reply:
Now we use artifact from built with flit